zcash_primitives/src/transaction/mod.rs

```diff
@@ -1029,7 +1130,63 @@ pub mod testing {
         }
     }
 
-    #[cfg(not(zcash_unstable = "zfuture"))]
+    #[cfg(all(not(zcash_unstable = "zfuture"), not(feature = "zip-233")))]
```

Comment:
Can this and the subsequent definitions not be merged? The conditionality seems to be regular and should not require repeating this much code.

Reply:
The `arb_txdata` functions are inside the `prop_compose!` macro. Moving the `cfg` conditions inside it causes compilation errors.